roles/girder-install/tasks/main.yml

```diff
@@ -0,0 +1,228 @@
+---
+
+  - name: apt | cache | update
+    apt: update_cache=yes
+    when: do_install|bool
+
+  - name: girder | deps | install
+    apt: name={{ item }} state=present update_cache=yes
+    with_items:
+      - apt-utils
+      - autoconf
+      - build-essential
+      - curl
+      - cmake
+      - g++
+      - gfortran
+      - git
+      - gzip
+      - libbz2-dev
+      - libcppunit-dev
+      - libexpat-dev
+      - libffi-dev
+      - libjpeg-dev
+      - libpng-dev
+      - libsqlite-dev
+      - libssl-dev
+      - libtool
+      - m4
+      - make
+      - mongodb
+      - ncurses-dev
+      - openjdk-7-jdk
+      - openjdk-7-jre-headless
```

Comment:
Why do we need gfortran, autoconf, sqlite, libexpat, mongodb, ncurses-dev, and openjdk?